src/lucky/force_ssl_handler.cr

```diff
@@ -46,7 +47,18 @@ class Lucky::ForceSSLHandler
 
   private def redirect_to_secure_version(context : HTTP::Server::Context)
     context.response.status_code = settings.redirect_status
+    add_hsts_if_enabled(context)
     context.response.headers["Location"] =
       "https://#{context.request.host}#{context.request.resource}"
   end
+
+  private def add_hsts_if_enabled(context : HTTP::Server::Context)
```

Comment:
Could you link to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security in a comment?